roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -100,13 +100,22 @@
     files:
     - "{{ tempdir }}/logging-elasticsearch-view-role.yaml"
     delete_after: true
-
 # configmap
+- fail:
+    msg: "The openshift_logging_es_log_appenders '{{openshift_logging_es_log_appenders}}' has an unrecognized option and only supports the following as a list: {{es_log_appenders | join(', ')}}"
+  when:
+    - es_logging_contents is undefined
+    - "{{ openshift_logging_es_log_appenders | list | difference(es_log_appenders) | length != 0 }}"
```

Comment:
couldn't we just have a separate assert that the provided options are valid? i think it would be better to fail and have the customer know they did something wrong that be surprised later to find out we did something they didn't plan on